src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -187,8 +187,8 @@ public sealed partial class TokenCache : ITokenCacheInternal
                         requestParams.RequestContext.ApiEvent.DurationInCacheInMs += sw.ElapsedMilliseconds;
                     }
 
-                    // Don't cache PoP access tokens from broker
-                    if (msalAccessTokenCacheItem != null && !(response.TokenSource == TokenSource.Broker && response.TokenType == Constants.PoPAuthHeaderPrefix))
+                    // Don't cache access tokens from broker
+                    if (msalAccessTokenCacheItem != null && !(response.TokenSource == TokenSource.Broker))
```

Comment:
To avoid re-testing Android / iOS, it would be safer to do this only on non-mobile